src/components/Pages/ListMushrooms/MushroomCard.tsx

```diff
@@ -2,10 +2,19 @@ import * as React from "react";
 import IconButton from "@mui/material/IconButton";
 import Tooltip from "@mui/material/Tooltip";
 import observationImage from "../../../assets/observationButton.svg";
-import extinctionImage from "../../../assets/extinctionButton.svg";
-import brazilImage from "../../../assets/brazil.svg";
-import vu from "../../../assets/vu.svg";
-import ts from "../../../assets/ts.svg";
+import tipoBrasileiro from "../../../assets/brasilT.png";
```

Comment:
É importante manter o padrão, todo restante do código estamos escrevendo em inglês, por que essa parte precisou ser em português? 

```suggestion
import brasilianType from "../../../assets/brasilT.png";
```